app/presenters/document_presenter.rb

```diff
@@ -122,10 +141,12 @@ def first_edition?
   end
 
   def filterable_metadata
-    {}
+    finder.facets.select{ |f| f.type == 'text' && f.filterable }
```

Comment:
See above re pushing methods into `Finder`. Not sure what this one would be though. `filterable_text_facets`? A bit clunky. Maybe `finder.text_facets.select(&:filterable?)`.